.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -14,7 +14,7 @@ assignees: ''
 
 <!--
 @babel/eslint-parser:
-  If you are having issues with JSX you might want to check out eslint-plugin-react. If there's an issue with new experimental syntax you might need check if it's supported by @babel/eslint-plugin.
+  If you are having issues with JSX you might want to check out eslint-plugin-react. If there's an issue with the new experimental syntax you might need to check if it's supported by @babel/eslint-plugin.
```

Comment:
We don't need the "the" since we're trying to address issues with _any_ new experimental syntax.

The "to" though, :+1:

```suggestion
  If you are having issues with JSX you might want to check out eslint-plugin-react. If there's an issue with new experimental syntax you might need to check if it's supported by @babel/eslint-plugin.
```